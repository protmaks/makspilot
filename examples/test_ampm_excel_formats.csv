DateTime,Event,Type
2022-01-01 10:41:52 AM,Meeting 1,With seconds
2022-01-01 10:41 AM,Meeting 2,Without seconds
1/1/2022 10:41:52 PM,Meeting 3,American format with seconds
1/1/2022 10:41 PM,Meeting 4,American format without seconds
1/1/22 10:41:52 AM,Meeting 5,Short year with seconds
1/1/22 10:41 AM,Meeting 6,Short year without seconds
2022-01-01 12:00:30 AM,Meeting 7,Midnight with seconds
2022-01-01 12:00 AM,Meeting 8,Midnight without seconds
2022-01-01 12:00:30 PM,Meeting 9,Noon with seconds
2022-01-01 12:00 PM,Meeting 10,Noon without seconds
