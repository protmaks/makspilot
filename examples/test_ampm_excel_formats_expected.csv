Event,DateTime,Type
Meeting 1,2022-01-01 10:41:52,Expected format
Meeting 2,2022-01-01 10:41:00,Expected format (no seconds = :00)
Meeting 3,2022-01-01 22:41:52,Expected format PM
Meeting 4,2022-01-01 22:41:00,Expected format PM (no seconds = :00)
Meeting 5,2022-01-01 10:41:52,Expected format
Meeting 6,2022-01-01 10:41:00,Expected format (no seconds = :00)
Meeting 7,2022-01-01 00:00:30,Expected format midnight
Meeting 8,2022-01-01 00:00:00,Expected format midnight (no seconds = :00)
Meeting 9,2022-01-01 12:00:30,Expected format noon
Meeting 10,2022-01-01 12:00:00,Expected format noon (no seconds = :00)
